packages/react-utils/src/hooks/useClickCooldown.ts

```diff
@@ -0,0 +1,28 @@
+import { useCallback, useRef, useState } from 'react';
+
+export const useClickCooldown = (cooldownMs: number) => {
```

Comment:
I would set a default value for this function and delete constants.

```suggestion
export const useClickCooldown = (cooldownMs: number = 3000) => {
```

Reply:
yes, good idea